src/index.html

```diff
@@ -3,11 +3,344 @@
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page" id="home">
     <script type="text/javascript" src="scripts/main.js"></script>
+    <div class="page-content">
+      <header class="header" >
+        <div class="sticky-top">
+          <a href="#home" class="sticky-top__home-logo">
+            <img src="./images/logo.svg" alt="home logo">
+          </a>
+
+          <div class="burger">
+            <input
+              type="checkbox"
+              name="burger-toggler"
+              id="burger-toggler"
+              class="burger__toggle"
+            >
+            <label
+              for="burger-toggler"
+            >
+              <span class="burger__activator"></span>
+            </label>
+
+            <ul class="burger__list">
+              <li class="burger__item">
+                <a href="#our-story" class="burger__link">
+                  OUR STORY
+                </a>
+              </li>
+              <li class="burger__item">
+                <a href="#about-us" class="burger__link">
+                  ABOUT US
+                </a>
+              </li>
+              <li class="burger__item">
+                <a href="#technology" class="burger__link">
+                  TECHNOLOGY
+                </a>
+              </li>
+              <li class="burger__item">
+                <a href="#features" class="burger__link">
+                  FEATURES
+                </a>
+              </li>
+              <li class="burger__item">
+                <a href="#get-in-touch" class="burger__link">
+                  GET IN TOUCH
+                </a>
+              </li>
+            </ul>
+          </div>
+          <a href="#specs" class="sticky-top__link">
+            specs
+          </a>
+          <div class="sticky-top__switchers">
+            <a href="#switch-en" class="sticky-top__link">
+              en
+            </a>
+            <span>/</span>
+            <a href="#switch-ua" class="sticky-top__link">ua</a>
+          </div>
+
+          <a href="#buy" class="sticky-top__link sticky-top__link--buy">
+            buy
+          </a>
+        </div>
+
+        <div class="explain">
+          <div class="explain__text explain-text">
+            <h1 class="explain-text__title">
+              Futuristic Wireless Speaker
+            </h1>
+            <p class="explain-text__p">
+              Luna’s performance is balanced and smooth in all frequency ranges which makes the music both naturally pleasant and distinctly more layered.
+            </p>
+          </div>
+          <img
+            src="./images/a347db2baba6b808980fb0422ff7f7f2_original.png"
+            alt="Wireless Speaker image"
+            class="explain__image"
+          >
+        </div>
+
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#our-story" class="nav__link">OUR STORY</a>
```

Comment:
Краще модифікуй текст через CSS
```suggestion
              <a href="#our-story" class="nav__link">
                OUR STORY
              </a>
```